2 - RTB/Documentazione interna/Verbali/Interni/23-11-13.typ

```diff
@@ -0,0 +1,65 @@
+#import "/template.typ": *
+
+#show: project.with(
+  title: "Verbale 13/11/23",
+  docType: "verbale",        
+  partecipants: (
+    (name: "Banzato Alessio", role: "Verificatore"),
+    (name: "Carraro Riccardo", role: "Verificatore"),
+    (name: "Gardin Giovanni",  role: "Redattore"),
+    (name: "Nardo Silvio",   role: "Redattore"),
+    (name: "Oseliero Antonio",   role: "Amministratore"),
+    (name: "Todesco Mattia",  role: "Verificatore"),
```

Comment:
Ruoli non conformi: attenersi alla suddivisione ponendo come verificatori i verificatori desiderari, segnando il redattore, e assegnando gli altri membri ai rispettivi ruoli (esempio analista/amministratore) (in questo caso io che sono stato segnato come verificatore mi lasci verificatore, mentre ad esempio mattia in questo periodo è analista)

Reply:
spero di aver corretto anche se non so se ho compreso al 100%, comunque anche questo deve essere normato, e anche qui è colpa mia